selfdrive/car/toyota/values.py

```diff
@@ -1057,6 +1057,7 @@ class CAR:
   },
   CAR.RAV4_TSS2: {
     (Ecu.engine, 0x700, None): [
+      b'\0018966342V3000\000\000\000\000',
```

Comment:
This should be moved down and arranged in order with the other firmware already there. Also the format should be \x00\x00

Reply:
Those \000 is from my log file, looked into values.py I was confused by \000 \x00 also. 
I tried to change them into /x00 and OP still works!

I'll resubmit another pull with proper info!
Best Wishes to all of u!